src/components/Graphs/LineGraph/index.js

```diff
@@ -73,7 +73,7 @@ export default class LineGraph extends XYGraph {
 
         let xAxisHeight       = xLabel ? chartHeightToPixel : 0;
         let legendWidth       = legend.show && linesData.length > 1 ? this.longestLabelLength(data, legendFn) * chartWidthToPixel : 0;
-        let xLabelWidth       = this.longestLabelLength(data, xLabelFn) * chartWidthToPixel;
+
```

Comment:
Actually, It was not used as of now, so was giving warning on the console. But yes might be required while trimming the long labels.